src/components/PupilViews/PupilResults/PupilResults.view.tsx

```diff
@@ -42,19 +40,22 @@ const QuizSectionRender = (props: QuizResultsProps) => {
       key={index}
       $gap={"all-spacing-5"}
     >
-      <QuizResultInner
-        index={index}
-        questionResult={questionResult}
-        quizArray={quizQuestionArray}
-        lessonSection={lessonSection}
-      />
-      {index !== quizQuestionArray.length - 1 && (
-        <OakHandDrawnHR
-          $height={"all-spacing-1"}
-          $pl={"inner-padding-xl"}
-          $ml={"space-between-s"}
+      <OakFlex $pb={["inner-padding-xl", "inner-padding-none"]}>
+        <QuizResultInner
+          index={index}
+          questionResult={questionResult}
+          quizArray={quizQuestionArray}
+          lessonSection={lessonSection}
         />
-      )}
+      </OakFlex>
+      <OakHandDrawnHR
```

Comment:
why have you changed the method here ? this feels like a regression on the previous